File: 3582-FindIndicesOfStableMountains/3582-FindIndicesOfStableMountains.java
// Last updated: 3/16/2026, 9:23:05 PM
class Solution {
    public List<Integer> stableMountains(int[] height, int threshold) {
        List<Integer> result = new ArrayList<>();
        int n = height.length;
        for(int index = 1 ; index < n ; index++){
            if(height[index-1] > threshold) result.add(index);
        }
        return result;
    }
}

File: 3582-FindIndicesOfStableMountains/3582-FindIndicesOfStableMountains.cpp
// Last updated: 3/17/2026, 5:30:55 AM
class Solution {
public:
    vector<int> stableMountains(vector<int>& height, int threshold) {
        vector<int> result;
        int n = height.size();
        for(int index = 1 ; index < n ; index++){
            if(height[index-1]>threshold) result.push_back(index);
        }
        return result;
    }
};